internals/models.py

```diff
@@ -474,6 +480,15 @@ class Feature(DictModel):
 
   DEFAULT_CACHE_KEY = 'features'
 
+  def __init__(self, *args, **kwargs):
+    # Initialise Feature.blink_components with a default value
+    if 'name' in kwargs:        # If name is present in kwargs then it would mean constructor is being called for creating a 
+                                # new feature rather than for fetching an existing feature. 
+      if 'blink_components' not in kwargs:
+        kwargs['blink_components'] = [BlinkComponent.DEFAULT_COMPONENT]
+
+    super(Feature, self).__init__(*args, **kwargs)
+
```

Comment:
This change is for assigning the default value to the property `blink_components`

Reply:
This looks good.